clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/TriggerAdapter.kt

```diff
@@ -101,6 +101,8 @@ class TriggerAdapter(triggerJSON: JSONObject) {
      */
     val profileAttrName: String? = triggerJSON.optString(Constants.KEY_PROFILE_ATTR_NAME, null)
 
+    val firstTimeOnly: Boolean = triggerJSON.optBoolean(Constants.KEY_FIRST_TIME_ONLY, false)
```

Comment:
move all constants of this class in companion object of this class itself so its readable.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/c3b509c964687b6bad9e9a21c8c2db2af654ab0a